hokusai/cli/review_app.py

```diff
@@ -18,12 +18,18 @@ def review_app(context_settings=CONTEXT_SETTINGS):
 @review_app.command(context_settings=CONTEXT_SETTINGS)
 @click.argument('app_name', type=click.STRING)
 @click.option('-v', '--verbose', type=click.BOOL, is_flag=True, help='Verbose output')
-@click.option('-sf', '--source-file', type=click.STRING, default='hokusai/staging.yml', help="The source yaml file from which to create the new resource file (default: hokusai/staging.yml)")
+@click.option(
+  '-sf',
+  '--source-file',
+  type=click.STRING,
+  default='hokusai/staging.yml',
+  help="The source yaml file from which to create the new resource file (default: hokusai/staging.yml)"
+)
 def setup(app_name, verbose, source_file):
-  """Setup a new review-app - create a Yaml file based on APP_NAME and --source-file"""
+  """Setup a new Review App"""
   set_verbosity(verbose)
   command(
-    hokusai.create_new_app_yaml,
+    hokusai.setup_review_app,
```

Comment:
since `setup` now does more than just creating yaml - it creates namespace as well.